hudi-common/src/test/java/org/apache/hudi/avro/TestHoodieAvroReaderContext.java

```diff
@@ -182,6 +185,20 @@ void getRecordKeyFromMetadataFields() {
     assertEquals(recordKey, avroReaderContext.getRecordKey(indexedRecord, schemaWithMetaFields));
   }
 
+  @ParameterizedTest
+  @ValueSource(booleans = {false, true})
+  void getKeyGeneratorClassDefaults(boolean isPartitioned) {
```

Comment:
Test MDT keygen too.

Reply:
This falls into the same bucket as the keygenerator based right? the MDT keygen is not available in this package and I don't feel comfortable moving the packages around

Reply:
This is no longer relevant.